src/js/index.d.ts

```diff
@@ -40,6 +40,8 @@ export * from './components/Markdown';
 export * from './components/MaskedInput';
 export * from './components/Menu';
 export * from './components/Meter';
+export * from './components/Page';
+export * from './components/PageSection';
```

Comment:
So, is it "Content" or "Section"?

Reply:
Oops is should be Content. Going back and fixing